usr/share/rear/pack/Linux-i386/300_copy_kernel.sh

```diff
@@ -10,7 +10,13 @@
 # Using another kernel is a TODO for now
 
 if [ ! -s "$KERNEL_FILE" ]; then
-    if [ -r "/boot/vmlinuz-$KERNEL_VERSION" ]; then
+    # add slackware test on top to prevent error on get_kernel_version
+    if [ -f /etc/slackware-version ]; then
```

Comment:
The test for Slackware should be done via the usually used variables
OS_MASTER_VENDOR OS_VENDOR OS_VENDOR_VERSION and so on,
see the SetOSVendorAndVersion function in usr/share/rear/lib/config-functions.sh
how those variables are set.

Perhaps those variables are already set right for Slackware by that function
perhaps you need to add support to get them set right for Slackware to that function.
To find out to what values those variables are set in your particular case
run `rear dump` for example in my case on SLES15 I get (excerpts):
<pre>
# usr/sbin/rear -D dump
...
This is a 'Linux-x86_64' system, compatible with 'Linux-i386'.
System definition:
                                    ARCH = Linux-i386
                                      OS = GNU/Linux
                        OS_MASTER_VENDOR = SUSE
                       OS_MASTER_VERSION = 15
                   OS_MASTER_VENDOR_ARCH = SUSE/i386
                OS_MASTER_VENDOR_VERSION = SUSE/15
           OS_MASTER_VENDOR_VERSION_ARCH = SUSE/15/i386
                               OS_VENDOR = SUSE_LINUX
                              OS_VERSION = 15
                          OS_VENDOR_ARCH = SUSE_LINUX/i386
                       OS_VENDOR_VERSION = SUSE_LINUX/15
                  OS_VENDOR_VERSION_ARCH = SUSE_LINUX/15/i386
</pre>

For example how those variables are used see the get_part_device_name_format function
in usr/share/rear/lib/layout-functions.sh that contains (excerpt):
<pre>
            case $OS_MASTER_VENDOR in
                (SUSE)
                    ...
                ;;
                (Fedora)
                    ...
                ;;
                (Debian)
                    ...
                ;;
                (*)
                    ....
                ;;
            esac
</pre>